pygfx/renderers/wgpu/_renderer.py

```diff
@@ -464,13 +465,28 @@ def render(
 
         # Get the list of objects to render, as they appear in the scene graph
         wobject_list = []
-        scene.traverse(wobject_list.append, True)
+
+        # todo: We can use `scene.background` to process the background separately
+        background = None
+
+        def _get_wobjects(obj):
+            if isinstance(obj, Background):
+                nonlocal background
+                background = obj
+            else:
+                wobject_list.append(obj)
+
+        scene.traverse(_get_wobjects, True)
```

Comment:
I don't really see the point for this change. It should not matter when the background is rendered since the depth test will take care of it. And even if it would not, it's easy for the user to just put the background first in the scene.

Reply:
When a cube box is used to represent the Skybox object (rather than just a plane), we can't simply set the depth of each vertex to 0.99999 in the shader. 
For this cube box, we need to ensure that it is drawed behind all objects without depth testing or writing its depth information (it won't obscure any object). 
So we need to make sure that it is drawn first. Users can do this easily (by putting the background first in the scene), but we can't guarantee that users will do this.